static/app/views/issueDetails/streamline/solutionsSection.spec.tsx

```diff
@@ -0,0 +1,143 @@
+import {EventFixture} from 'sentry-fixture/event';
+import {GroupFixture} from 'sentry-fixture/group';
+import {OrganizationFixture} from 'sentry-fixture/organization';
+import {ProjectFixture} from 'sentry-fixture/project';
+
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import {useGroupSummary} from 'sentry/components/group/groupSummary';
+import {getConfigForIssueType} from 'sentry/utils/issueTypeConfig';
+import useOrganization from 'sentry/utils/useOrganization';
+import SolutionsSection from 'sentry/views/issueDetails/streamline/solutionsSection';
+
+jest.mock('sentry/components/group/groupSummary');
+jest.mock('sentry/utils/useOrganization');
+jest.mock('sentry/utils/issueTypeConfig');
+
+describe('SolutionsSection', () => {
+  const mockEvent = EventFixture();
+  const mockGroup = GroupFixture();
+  const mockProject = ProjectFixture();
+  const {organization} = initializeOrg({
+    organization: OrganizationFixture(),
+  });
+
+  beforeEach(() => {
+    (getConfigForIssueType as jest.Mock).mockReturnValue({
+      issueSummary: {
+        enabled: true,
+      },
+      resources: {
+        description: 'Test Resource',
+        links: [{link: 'https://example.com', text: 'Test Link'}],
+        linksByPlatform: {},
+      },
+    });
+
+    (useOrganization as jest.Mock).mockReturnValue({
+      ...organization,
+      hideAiFeatures: false,
+      genAIConsent: true,
+    });
+  });
+
+  const renderComponent = () =>
+    render(
+      <SolutionsSection event={mockEvent} group={mockGroup} project={mockProject} />,
+      {
+        organization,
+      }
+    );
+
+  it('renders loading state when summary is pending', () => {
+    (useGroupSummary as jest.Mock).mockReturnValue({
+      isPending: true,
+      data: null,
+    });
```

Comment:
mock the api call and not the hook directly
```suggestion
    (useGroupSummary as jest.Mock).mockReturnValue({
      isPending: true,
      data: null,
    });
```